fleet_vehicle_log_fuel/__manifest__.py

```diff
@@ -4,17 +4,17 @@
 {
     "name": "Fleet Vehicle Log Fuel",
     "summary": "Add Log Fuels for your vehicles.",
-    "version": "15.0.2.0.0",
-    "category": "Human Resources/Fleet",
-    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "version": "16.0.1.0.0",
+    "category": "Fleet",
+    "author": "ForgeFlow, Tecnativam, Odoo Community Association (OCA)",
```

Comment:
We don't plan for now to rebrand our company, hehe

```suggestion
    "author": "ForgeFlow, Tecnativa, Odoo Community Association (OCA)",
```